km52/Kholin_Nikita/laboratory1.sql

```diff
@@ -1,2 +1,114 @@
 -- LABORATORY WORK 1
 -- BY Kholin_Nikita
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+видаляти дані з таблиці
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+create user kholin identified by kholin_password;
```

Comment:
add tablespaces and first line without ";"
create user kholin identified by kholin_password
default tablespace "users"
temporary tablespace "temp";